lib/src/common/disposable.dart

```diff
@@ -417,9 +417,8 @@ class Disposable implements _Disposable, DisposableManagerV7, LeakFlagger {
 
     if (_debugMode) {
       stopwatch.stop();
-      _logger
-          .info('$runtimeType $hashCode took ${stopwatch.elapsedMicroseconds /
-          1000000.0} seconds to dispose');
+      var t = stopwatch.elapsedMicroseconds / 1000000.0;
+      _logger.info('$runtimeType $hashCode took $t seconds to dispose');
```

Comment:
broke this into 2 lines so formatting wouldn't be different between dart 1 and dart 2 🤷‍♂️